### PR TITLE
Update stix2.1 mapping files in azure sentinel module

### PR DESCRIPTION
**Pull request by mdazam1942**

### CODE REVIEW

File: stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/alert_from_stix_map.json [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+      "hashes.unknown": ["fileStates.fileHash.hashValue"]
+    }
+  },
+   "process" : {
+    "fields": {
+      "x_name": ["processes.name", "processes.parentProcessName"],
+      "parent_ref.name": ["processes.parentProcessName"],
+      "command_line": ["processes.commandLine"],
+      "pid": ["processes.processId", "processes.parentProcessId", "registryKeyStates.processId"],
+      "created": ["processes.createdDateTime"],
```
Comment (delliott90) [inline stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/alert_from_stix_map.json:59]: Should be `created_time` for 2.1

File: stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json [hunk 2]
```diff
@@ -1,598 +1,1356 @@
...
-      },
-      "wasRunning": {
-        "key": "x-msazure-sentinel-alert.malwareStates.wasRunning",
-        "object": "alert"
+    "commandLine": {
+      "key": "process.command_line",
+      "object": "process"
+    },
+    "createdDateTime": {
+      "key": "process.created",
```
Comment (delliott90) [inline stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json:370]: Should be `process.created_time` for 2.1

File: stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json [hunk 3]
```diff
@@ -1,598 +1,1356 @@
...
+    "isElevated": {
+      "key": "processes.x_isElevated",
+      "object": "process"
+    },
+    "name": {
+      "key": "process.x_name",
+      "object": "process"
+    },
+    "parentProcessCreatedDateTime": {
+      "key": "process.created",
```
Comment (delliott90) [inline stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json:424]: `created_time`

File: stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json [hunk 4]
```diff
@@ -1,598 +1,1356 @@
...
-        "object": "dir"
+      "processCommandLine": {
+          "key": "process.command_line",
+          "object": "process"
       },
-      "processId": {
-        "key": "process.pid",
-        "object": "process"
+      "processCreationDateTime": {
+          "key": "process.created",
```
Comment (delliott90) [inline stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json:966]: created_time

File: stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json [hunk 5]
```diff
@@ -1,598 +1,1356 @@
...
+          "object": "process"
+      },
+      "parentProcessCreationDateTime": {
+          "key": "process.created",
```
Comment (delliott90) [inline stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json:970]: created_time

File: stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/alert_from_stix_map.json (hunk 1 above)
Comment (mdazam1942) [reply stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/alert_from_stix_map.json:59]: fixed

File: stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json (hunk 2 above)
Comment (mdazam1942) [reply stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json:370]: fixed

File: stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json (hunk 3 above)
Comment (mdazam1942) [reply stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json:424]: fixed

File: stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json (hunk 4 above)
Comment (mdazam1942) [reply stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json:966]: fixed

File: stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json (hunk 5 above)
Comment (mdazam1942) [reply stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json:970]: fixed

**Review by delliott90 — APPROVED** [review]

**Pull request merged**